packages/ember-testing/lib/helpers.js

```diff
@@ -278,8 +278,17 @@ asyncHelper('visit', visit);
   });
   ```
 
+  Accepts `context` as an optional second argument. Useful for scoping the element lookup.
+
+  ```javascript
+  click('.some-jQuery-selector', '.modal-dialog').then(function() {
```

Comment:
Maybe the second argument should be an element instead of a selector string? This could be rewritten as:

```
click('.modal-dialog .some-jQuery-selector').then(function() {
```

Which is shorter and clearer. I actually don't know the use of the second argument so it's hard for me to suggest an atlernative. @rwjblue @locks Any ideas?